src/calibration.cpp

```diff
@@ -135,6 +135,16 @@ void Calibration::cancelCalibration()
 	m_cancel = true;
 }
 
+bool Calibration::hasContextCalibration()
```

Comment:
Can this method be const? Does it modify the state of the Calibration object?

Reply:
fixed